### PR TITLE
PROGRAMMES-6754 - hide barkers and off-airs

### PR DESCRIPTION
**Pull request by Bobkidbob**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6754

### CODE REVIEW

**Comment by Bobkidbob** [comment]
Test this please.

File: src/Data/ProgrammesDb/Entity/Programme.php [hunk 1]
```diff
@@ -129,4 +136,14 @@ public function setFirstBroadcastDate(?DateTime $firstBroadcastDate): void
     {
         $this->firstBroadcastDate = $firstBroadcastDate;
     }
+
+    public function getIsGap(): bool
```
Comment (cisco87) [inline src/Data/ProgrammesDb/Entity/Programme.php]: AFAIR (Remember :D) doctrine also tries (isGap()) ? Which I think might be better? No?

Comment (BjornTwachtmann) [reply src/Data/ProgrammesDb/Entity/Programme.php]: We use getIs...() everywhere else, Doctrine isn't doing any magic here, it's just a getter

Comment (cisco87) [reply src/Data/ProgrammesDb/Entity/Programme.php]: Alright got it, then better to make it consistent, sorry

**Pull request merged**